Makefile

```diff
@@ -49,6 +49,10 @@ lint:  ## Run linter
 	@go vet ./...
 
 
+.PHONY: docs
+docs:  ## Generate swagger docs
+	@go install github.com/swaggo/swag/cmd/swag@latest
```

Comment:
Can we pin this to `v1.16.2` instead of latest.

Reply:
> Can we pin this to `v1.16.2` instead of latest.

I just pushed a --fixup :)